src/reducers/url-state.js

```diff
@@ -223,9 +253,10 @@ const globalTrackOrder: Reducer<TrackIndex[]> = (state = [], action) => {
     case 'VIEW_PROFILE':
     case 'CHANGE_GLOBAL_TRACK_ORDER':
       return action.globalTrackOrder;
-    case 'PROFILE_PUBLISHED':
-      // This may no longer be valid because of PII sanitization.
-      return [];
+    case 'SANITIZE_PROFILE':
+      // If some threads were removed, do not even attempt to figure this out. It's
+      // complicated, and not many people use this feature.
```

Comment:
😄 (works for me though :))

Reply:
Yeah, I'd like to remove all the ordering logic. It really complicates things.